### ISSUE TITLE
Web component: Content group simple - no alt text for this graphic

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4410]
## Detailed description
On voiceover iPhone & JAWS FF

Issue:  Content group simple - no alt text for this graphic

Expected : Screen reader should announce a descriptive alt text for the graphic. The alt text can be given using alt attribute

**Comment by annawen1** [comment]
@Praveen-sr I tested this with Iphone VO and it's reading out the alt text as "Image Alt Text". Can this bug be tested again?

**Comment by Praveen-sr** [comment]
@annawen1 Re tested this on iPhone & JAWS FF, working fine now. 
Thanks!

CC @RobertaJHahn
